api/v1alpha1/bundle.go

```diff
@@ -20,6 +20,9 @@ const (
 	// BundleAPIVersionSelector is the CUE path for the Timoni's bundle API version.
 	BundleAPIVersionSelector Selector = "bundle.apiVersion"
 
+	// BundleMetadataName is the CUE path for the Timoni's bundle name.
+	BundleMetadataName Selector = "bundle.metadata.name"
```

Comment:
Given that Bundles are not Kubernetes CRDs and they don't have `.spec`, I think it's best to remove `.metadata` and have `name` as top level prop.

Reply:
Vouch for this as well, to prevent a Kustomize-like (and eksctl, etc.) scenario where people get confused about things looking like a Kubernetes API object, while they actually aren't.